src/styles/blocks/product-catalog.scss

```diff
@@ -0,0 +1,29 @@
+.product-catalog {
+  padding-block: 50px;
+  padding-inline: 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
The `grid-template-columns` property should have a `minmax` function to ensure that the grid items have a minimum and maximum width, as per the code style guide. Consider using `grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));` to allow the cards to grow and fill the available space.